src/main/java/gov/cms/madie/dto/qrda/StratificationDTO.java

```diff
@@ -0,0 +1,18 @@
+package gov.cms.madie.dto.qrda;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder(toBuilder = true)
+public class StratificationDTO {
```

Comment:
If these DTOs are going to be shared across multiple services we should probably eventually move them to madie-java-models or something. But that can be done later

Reply:
Agreed. I should make a clean-up task to do that once the changes clear QA.